src/components/ChangePlan/Checkout/Checkout.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import HeaderSection from '../../shared/HeaderSection/HeaderSection';
+import { useIntl } from 'react-intl';
+import { Loading } from '../../Loading/Loading';
+import { Helmet } from 'react-helmet';
+import { ContactInformation } from './ContactInformation/ContactInformation';
+import { getCountries } from '../../../utils';
+
+const Checkout = ({ dependencies: { dopplerUserApiClient } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const selectOption = _('checkoutProcessForm.empty_option_select');
+
+  const checkoutSteps = {
+    contactInformation: 'contact-information',
+    billingInformarion: 'billing-information',
+    paymentInformarion: 'payment-information',
+  };
+
+  const [activeStep, setActiveStep] = useState(checkoutSteps.contactInformation);
+  const [state, setState] = useState({ loading: true });
+
+  const submitContactInformationForm = async (values) => {
+    const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+
+    if (result.success) {
+      setActiveStep(checkoutSteps.contactInformation);
+    }
+
+    return result;
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+      const defaultValue = { key: '', value: selectOption };
+      const countries = [defaultValue, ...getCountries(intl.locale)];
```

Comment:
In my opinion, these things (`selectOption`, `countries`, `intl` and `getCountries`) has nothing to do inside the `useEffect` am I missing something?

Reply:
I added thes becuase I had same warnings. For example:

```src\components\ChangePlan\Checkout\Checkout.js
  Line 52:6:  React Hook useEffect has a missing dependency: 'selectOption'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```

Maybe was the wrong way to fix it.

Reply:
Because in my opinion, it should be outside `useEffect`, let me know if you cannot fix it.